ground/Ground-00.ipynb

```diff
@@ -30,14 +39,14 @@
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 2
+    "version": 3
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython2",
-   "version": "2.7.13"
+   "pygments_lexer": "ipython3",
```

Comment:
Is this intentional?

Reply:
(Seems at odds with the description that this is a cosmetic PR)

Reply:
I think this might have been a local configuration of Jupyter issue? It looks like my env. changed it back to the previous configuration [in the next PR](https://github.com/ucbrise/risecamp/pull/45/files#diff-cc8fdff6ba32f417257decd7a5947845R39).